tests/acceptance/agents/test_signing_coordinator_agent.py

```diff
@@ -201,6 +211,9 @@ def test_post_signature(
     deployed_multisig = nucypher_dependency.ThresholdSigningMultisig.at(
         threshold_signing_multisig_clone_factory.getCloneAddress(cohort_id)
     )
-    assert deployed_multisig.getSigners() == cohort_operators
+    assert (
+        deployed_multisig.getSigners()
+        == accounts.unassigned_accounts[: len(cohort_operators)]
```

Comment:
Perhaps `accounts.unassigned_accounts[: len(cohort_operators)]` should be made into a fixture. Same for L49.

Reply:
✔️